src/Tools/BuildActionTelemetryTable/BuildActionTelemetryTable.csproj

```diff
@@ -27,6 +27,7 @@
     <PackageReference Include="Microsoft.VisualStudio.Shell.Framework" Version="$(MicrosoftVisualStudioShellFrameworkVersion)" NoWarn="NU1701" />
     <PackageReference Include="Microsoft.VisualStudio.GraphModel" Version="$(MicrosoftVisualStudioGraphModelVersion)" NoWarn="NU1701" />
     <PackageReference Include="Microsoft.VisualStudio.Imaging" Version="$(MicrosoftVisualStudioImagingVersion)" NoWarn="NU1701" />
+    <PackageReference Include="Microsoft.IO.Redist" Version="$(MicrosoftIORedistVersion)" NoWarn="NU1701"/>
```

Comment:
"Microsoft.VisualStudio.Shell.Framework" add a reference to this package sometime in 17.6, so it is generating NU1701.
This won't affect VS since the pacakge is excluded from our insertion list https://github.com/dotnet/roslyn/blob/3a7a7407ea3c831630ecf2754092c33df3a6e452/src/Setup/DevDivInsertionFiles/DevDivInsertionFiles.csproj#L150